### PR TITLE
refactor: Remove Right Menu

### LINKED ISSUE #179: 🔧 [Refactor] Menu
## :zap: Refactor ticket

We had a discussion about the menu. Currently we have two menus. There is two options:
- Have one menu
- Have two menus with distinctively different tasks

The common consensus was: Two menus, one for banking, one for the user. I disagree tho - I believe one menu is sufficient at the given time, its less work, easier to communicate to the user.

We could not reach a final decision. If I have to decide, I would go with 1 menu as stated above and I will outline it here. Please make a suggestion for the two menu option (@Mogge (?))

Furthermore I'd like to have symbols for every menu entry using fontawesome (or something else) as icon library. The corresponding icons are denoted behind the Menu. The English translation has to be found according to the German words.

Menu:
```
- Überweisen (Statt Senden) (money-bill-wave)
- Umsätze (statt Transactions) (wallet)
- Schöpfen (statt Aktivitäten) (hand-holding-medical)
----
- Stammdaten (Statt Mein Profil & Settings) (address-card)
----
- Ausloggen (sign-out-alt)
```

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
rechtes menü habe ich entfernt

### Issues
- fixes #179

### CODE REVIEW

File: frontend/src/routes/routes.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ const routes = [
   },
   {
     path: '/profile',
+    name: 'Profile',
```
Comment (ulfgebhardt) [inline frontend/src/routes/routes.js:13]: @Mogge do we use this? @ogerly How is this related to the PR?

File: frontend/src/views/Layout/DashboardNavbar.vue [hunk 2]
```diff
@@ -120,5 +70,10 @@ export default {
       this.$store.dispatch('logout')
     },
   },
+  beforeDestroy() {
+    if (this.$sidebar.showSidebar) {
+      this.$sidebar.showSidebar = false
+    }
```
Comment (ulfgebhardt) [inline frontend/src/views/Layout/DashboardNavbar.vue:76]: Isn't this solved by the two different layouts? Do we need those lines? What do they do?

**Review by ulfgebhardt — COMMENTED** [review]
Two questions. Also I do not understand why two menus are effected. The one in `Sidebar.vue` and the one in `DashboardNavbar.vue`

File: frontend/src/routes/routes.js (hunk 1 above)
Comment (Mogge) [reply frontend/src/routes/routes.js:13]: No, so far we do not have named views.

**Review by Mogge — APPROVED** [review]
